tests/features/versioned_branches_test.py

```diff
@@ -600,7 +591,25 @@ def test_fix_py3x_only_code(s, expected):
         '    3+7\n'
         'else:\n'
         '    3-6\n',
+
+        # this one *could* be fixed, it's just not implemented yet
+        # see test_no_else_py3x_only_code
+        'import sys\n'
+        'if sys.version_info >= (3, 6):\n'
+        '    pass',
     ),
 )
 def test_fix_py3x_only_noop(s):
     assert _fix_plugins(s, settings=Settings(min_version=(3, 6))) == s
+
+
+@pytest.mark.xfail(reason='not implemented')
+def test_no_else_py3x_only_code():
+
+    s = (
+        'import sys\n'
+        'if sys.version_info >= (3, 6):\n'
+        '    pass'
+    )
+    expected = 'import sys\n'
```

Comment:
I expect `import sys\npass\n` -- and my checkout of `pyupgrade` correctly makes that happen (?)

Reply:
you're right, sorry, it's getting late in 🇫🇷